hw/fpga/README.md

```diff
@@ -67,62 +74,42 @@ This script provides a number of configuration options for features that can be
     - Run Synthesis: `launch_runs synth_1`
     - [Optional] Set Up Debug signals on Synthesized Design
     - Run Implementation: `launch_runs impl_1`
-    - Generate Bitstream: `write_bitstream -bin_file \tmp\caliptra_fpga`
-
-### Loading and execution Steps: ###
-[setup_fpga.sh](setup_fpga.sh) performs platform setup that is needed after each boot.
- - Disables CPU IDLE. Vivado HW Manager access during IDLE causes crashes.
- - Reduces fan speed by setting the GPIO pin connected to the fan controller FULLSPD pin to output.
-   - https://support.xilinx.com/s/question/0D52E00006iHuopSAC/zcu104-fan-running-at-max-speed?language=en_US
- - Builds and installs the rom_backdoor and io_module kernel modules.
- - Sets the clock for the FPGA logic.
- - Installs the provided FPGA image.
-
+    - Generate Device Image: `write_device_image $outputDir/caliptra_fpga`
+    - Export hardware: `write_hw_platform -fixed -include_bit -force -file $outputDir/caliptra_fpga.xsa`
+
+### Build boot.bin: ###
+ - Source petalinux settings.sh
+ - Run steps from [create_boot_bin.sh](create_boot_bin.sh) to use Petalinux to create the BOOT.BIN
+   - `./create_boot_bin.sh /path/to/caliptra_fpga_project_bd_wrapper.xsa`
+ - Copy petalinux_project/images/linux/BOOT.BIN to the boot partition as boot1900.bin
+   - If the Ubuntu image is booted, it will mount the boot partition at /boot/firmware/
+   - If boot1900.bin fails to boot the system will fallback to the default boot1901.bin
+
+### Running Caliptra tests from the FPGA: ###
 ```shell
-sudo ./hw/fpga/setup_fpga.sh caliptra_fpga.bin
+sudo apt update
```

Comment:
Missing a `git clone https://github.com/chipsalliance/caliptra-sw.git` I think (and `git submodule init && git submodule update` once you're in the right branch).